src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      const bookId = action.payload;
+      state.books.concat(bookId);
+    },
+    removeBook: (state, action) => {
+      const bookId = action.payload;
+      state.books.filter((book) => book.item_id !== bookId);
+    },
+  },
```

Comment:
- [ ] Please the `addBook`  and `removeBook` methods are not working at the level of the User Interface try to fix that.

- **Adding and Removing actions must be implemented according to project requirements.**

Reply:
Hi, I think this is not in the requirement

Reply:
![prove](https://user-images.githubusercontent.com/110599399/221986084-8e2d37a6-88ca-4295-aa45-96f65ce0e1a6.JPG)

Reply:
Hi @RaphDasilva 

You are right I use the old project requirement.
Please your dashboard will be updated soon.